cpp/src/IceGrid/Grammar.y

```diff
@@ -99,6 +99,7 @@ yyerror(const char* s)
 %token ICEGRID_STRING
 %token ICEGRID_START
 %token ICEGRID_STOP
+%token ICEGRID_STATUS
```

Comment:
The existing command for server "status" is called state.

We should consider renaming state -> status (as in server status), and possibly keeping state as an alias for status.